src/engine/strat_engine/engine.rs

```diff
@@ -76,6 +80,12 @@ impl StratEngine {
             return Err(EngineError::Engine(ErrorEnum::Error, err_msg));
         }
 
+        if let Err(err) = fs::create_dir(devlinks::DEV_PATH) {
```

Comment:
If we're doing it here, we don't need to do it in setup_devlinks. (not that it hurts, it's just duplicate.) So you could remove it there. 

A step further would be to add another little function to devlinks.rs that does this and that you call from here, if you think it's worth it to keep all the devlink stuff in one spot. Hmm, I guess we can always do that later if the code becomes more significant.

Reply:
Huh? We need it set up, we should set it up in one place, not two, not zero.